# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/printing/content/printUtils.js
# Commit: 909001ac06d8
# Full Hash: 909001ac06d8b3982116a0b0f753ae365b1cea22
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-07-23 03:20:25
# Regressor Bug: 1652270
# File Overlap Count: 1
# Description:
#   Bug 1652270. Convert nsFrameLoader::Print to return a Promise. r=farre,remote-protocol-reviewers,marionette-reviewers,jgraham,whimboo,mixedpuppy
#   
#   (Instead of requiring callers to pass an nsIWebProgressListener.)
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83249
# ==============================================================================

diff -r 8cfbd5213d04 -r 909001ac06d8 toolkit/components/printing/content/printUtils.js
--- a/toolkit/components/printing/content/printUtils.js	Thu Jul 23 02:01:54 2020 +0300
+++ b/toolkit/components/printing/content/printUtils.js	Wed Jul 22 23:17:45 2020 +0000
@@ -182,6 +182,8 @@
       printSettings.docURL = this._originalURL;
     }
 
+    // At some point we should handle the Promise that this returns (report
+    // rejection to telemetry?)
     topBrowser.print(windowID, printSettings);
 
     if (printPreviewIsOpen) {